development/request_guidelines.md

```diff
@@ -1,3 +1,21 @@
+# Pull Request Expectations
+
+We have a set of norms around pull requests and merging of code. Most of these have evolved in an informal way among development teams or during our retrospectives while discussing things we could improve. There are exceptions to many rules, but this document attempts to identify our baseline expectations.
+
+## For Submitters
+
+- When containers are modified as a part of a pull request manifests should be updated to use those containers. Alternatively, instructions may be provided to inform reviewers how to deploy the change.
+- All non-trivial pull requests are expected to have a set of testing manual instructions with them. The more information you can provide reviewers about how to verify the change the more likely you are to get good reviews.
```

Comment:
Grammer: s/testing manual/manual testing/

Reply:
Grammar + suggestion: "The more information you can provide to the reviewer(s) about how to verify the change, the more likely it is to get prioritized and the more likely you'll get detailed feedback."

Reply:
grammar: s/grammer/grammar/g